sdk/src/main/java/com/horizen/account/secret/PrivateKeySecp256k1.java

```diff
@@ -71,7 +68,8 @@ public boolean owns(ProofOfKnowledgeProposition proposition) {
 
     @Override
     public SignatureSecp256k1 sign(byte[] message) {
-        return Secp256k1.sign(privateKey, message);
+        Secp256k1.Signature signature = Secp256k1.sign(privateKey, message);
+        return new SignatureSecp256k1(signature.v, signature.r, signature.s);
```

Comment:
What is the reason behind this change?

Reply:
To make `Secp256k1` class agnostic to SDK types structure. One of them is `SignatureSecp256k1`